docs/tutorials/podman_pasta_ipv6_with_snid_ipv4.md

```diff
@@ -0,0 +1,761 @@
+
+The Podman Host (Bare Metal or e.g. KVM Virtual Machine) is supposed to have:
+- Public IPv4 Address: `198.51.100.10` (https://www.rfc-editor.org/rfc/rfc5737)
+- Private IPv4 Address: `172.16.1.10/24`
+- Public+Private IPv6 Address: IPv6: `2001:db8:0000:0001:0000:0000:0000:0100/128` (https://www.rfc-editor.org/rfc/rfc3849.html)
+
+Each Application will furthermore have an IPv6 Address, to which it will bind the Required Ports, which are typically:
+- Port `443/tcp` (HTTPS)
+- Port `443/udp` (HTTP3)
+- Port `80/tcp` (HTTP)
+
+The Applications are supposed to be located within the following Network (corresponding to `snid` Backend CIDR Configuration): `2001:db8:0000:0001:0000:0000:0001:0000/112` (`2001:db8:0000:0001:0000:0000:0001:0000` ... `2001:db8:0000:0001:0000:0000:0001:ffff`). Other IPv6 Addresses are also Possible, but then you must adjust `snid` Backend CIDR accordingly !
+
+For simplicity, the Application (`application01.MYDOMAIN.TLD`) described in this Tutorial will have `APPLICATION_IPV6_ADDRESS="2001:db8:0000:0001:0000:0000:0001:0001"`.
+
+One Remote End-Client (Laptop) is supposed to have IPv4 Address:
+- Public IPv4 Address: `192.0.2.100` (https://www.rfc-editor.org/rfc/rfc5737)
+
+Another Remote End-Client (Laptop) is supposed to have IPv6 Address:
+- Public+Private IPv6 Address: `2001:db8:2222:2222:0000:0000:0000:0100/128` (https://www.rfc-editor.org/rfc/rfc3849.html)
+
+The IP Addresses of the Routers/Firewalls themselves are not described in this Section, as they are not relevant for the Configuration described by this Tutorial.
+
+> **Note**  
+>
+> Remember to Open the Required Ports in the Upstream Firewall (e.g. OPNSense, OpenWRT, etc) as well as on the Podman Host, if a Firewall is Enabled (e.g. `firewalld` , `ufw` , `iptables` , `nftables`, etc).
+
+# snid Setup
+In order to ensure that IPv4-only Remote Clients can access the Applications running on the Podman Host (Applications which ONLY have an IPv6 Address), an IPv4 <-> IPv6 Translation must be performed.
+
+This Setup assumes that `snid` is installed on the Podman Host itself.
+
+Other Setups where `snid` is running e.g. in a separate KVM Virtual Machine are possible, but require setting up a Static Route from the Podman Host to `64:ff9b:1::/96` (otherwise the Application can be contacted by the Remote Client, but the Application will NOT be able to send any Reply to it).
+
+The easiest way to run `snid` is to download the precompiled Binary from the Official Website and setup a Systemd Service for it. Compiling `snid` from Source it's possible but it involves installing the `go` Development Toolchain, which is NOT the purpose of this Tutorial.
+
+First Download the Program:
+```
+# Run these Commands on the Podman Host as <root>
+mkdir -p /opt/snid
+wget https://github.com/AGWA/snid/releases/download/v0.3.0/snid-v0.3.0-linux-amd64 -O /opt/snid/snid
+chmod +x /opt/snid/snid
+```
+
+## Rootfull Service
```

Comment:
Use either the rootful or the rootless, remove the other.

Reply:
I removed the Rootfull Service. The (split) Rootless Service can anyways be easily converted to Root.